### ISSUE TITLE
autoRemoveInactiveDevices not working

### ISSUE THREAD

**Issue by ahdung** [Ylianst/MeshCentral#6268]
**Describe the bug**
`autoRemoveInactiveDevices` not working.

**Screenshots**
![微信截图_20240723111733](https://github.com/user-attachments/assets/850cd0ec-d481-4724-a0ae-bf3a987edde3)

**Server Software (please complete the following information):**
 - OS: windows server 2022, 2008R2 too
 - Virtualization: no
 - Network: WAN
 - Version: 1.1.24, BTW "Check server version" need a long time show "Loading..."
 - Node: 18.19.0

**Your config.json file**
```
{
   "settings": {
      "cert": ***,
      "WANonly": true,
      "port": ***
      "redirPort": 0,
      "mpsPort": 0,
      "webRTC": true,
      "_tlsOffload": false,
      "_mpsTlsOffload": true,
      "_trustedProxy": ***
      "_minify": true,
      "noAgentUpdate": 1,
      "ignoreAgentHashCheck":true
   },
   "domains": {
      "": {
         "autoRemoveInactiveDevices": 1
      }
   }
}
```

**Comment by si458** [comment]
oh i think ive found the bug in the code! you cant set 1 , it must between 2 and 2000 !?
```
var v = parent.config.domains[i].autoremoveinactivedevices;
if ((v > 1) && (v <= 2000)) {
  if (v < minRemoveInactiveDevice) { minRemoveInactiveDevice = v; }
  removeInactiveDevicesPerDomain[i] = v;
  minRemoveInactiveDevicesPerDomain[i] = v;
}
```
dont suppose you can test by changing `db.js` line 139 to
`if ((v => 1) && (v <= 2000)) {`

**Comment by si458** [comment]
all fixed, will be ready for `master` docker image in about 15mins
and also npm develop build in 15mins too `npm install Ylianst/MeshCentral`

**Comment by stavros-k** [comment]
> `if ((v => 1) && (v <= 2000)) {`

Shouldn't this be '>=' instead of '=>'?

**Comment by si458** [comment]
@stavros-k oh FFS, its one of those days !?

**Comment by ahdung** [comment]
that's funny, Thank you very much.

**Comment by si458** [comment]
Correctly fixed in 1.1.27 :)
